README.md

```diff
@@ -271,6 +347,12 @@ To clean up the stopped containers, run:
 docker compose down --volumes
 ```
 
+### Build and Run
```

Comment:
![Screenshot 2025-05-12 at 8 39 59 AM](https://github.com/user-attachments/assets/95413f3d-360e-4946-9c32-6f3f4cc7dc59)

There is context that was dropped here. While some of it is a repeat for the time being keep all of what was there before with podman vs docker commands